src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld_Tick.java

```diff
@@ -67,10 +67,12 @@ private boolean canUpdateEntity(World _this, int startBlockX, int oldStartBlockY
 		if (!this.isCubicWorld()) {
 			return isAreaLoaded(startBlockX, oldStartBlockY, startBlockZ, endBlockX, oldEndBlockY, endBlockZ, allowEmpty);
 		}
+
+		BlockPos entityPos = new BlockPos(updateEntity_entityPosX, updateEntity_entityPosY, updateEntity_entityPosZ);
+
 		return this.isRemote() ||
-				this.getCubeFromBlockCoords(
-						new BlockPos(updateEntity_entityPosX, updateEntity_entityPosY, updateEntity_entityPosZ))
-						.getTickets().shouldTick();
+			  !this.isValid(entityPos) || // if an entity is below the world we want it to keep ticking
+			   this.getCubeFromBlockCoords(entityPos).getTickets().shouldTick();
```

Comment:
Without any buffer radius bigger entities will still load chunks because of physics calculations